xbmc/pvr/PVRManager.cpp

```diff
@@ -727,7 +730,9 @@ bool CPVRManager::UpdateComponents(ManagerState stateToCheck,
     return false;
   }
 
-  PublishEvent(PVREvent::ChannelGroupsLoaded);
+  // defer until after Reinit playbackstate in Process()
```

Comment:
Remove this comment as it refers to another implementation detail and thus must be changed whenever this detail might change in the future, which easily can be forgotten.